tools/mac_setup.sh

```diff
@@ -28,6 +33,24 @@ elif [[ $SHELL == "/bin/bash" ]]; then
   RC_FILE="$HOME/.bash_profile"
 fi
 
+# Build requirements for macOS
+# https://github.com/pyenv/pyenv/issues/1740
+# https://github.com/pyca/cryptography/blob/main/docs/installation.rst
+rustup-init -y
+
+if [ -n "$RC_FILE" ]; then
+  echo 'export LDFLAGS="$LDFLAGS -L/usr/local/opt/zlib/lib"' >> $RC_FILE
```

Comment:
it's not great to spam a user's bashrc. does brew not handle this for us?

Reply:
Unfortunately not. An alternative would be to only define the flags for `pyenv install`. `openssl` and `rust` would need to stay probably.